backend/json.go

```diff
@@ -0,0 +1,93 @@
+package backend
+
+import (
+	jsoniter "github.com/json-iterator/go"
+)
+
+// // This will make sure jsoniter uses a fast JSON serialization strategy
+// var (
+// 	_ = initEncoders()
```

Comment:
Isn't this exactly the use case for an `init` function? This just looks more convoluted.

I don't have any opinion on registering custom codecs, is it for speed? Is there any problem with it?

Reply:
@aknuds1 -- ah ok, I avoided init() because of the lint rules, but if that seems like a reasonable option then lets do it.

> is it for speed?

Yes -- one of the biggest ways that jsoniter speeds things up is by writing to the same stream throught the whole marshall process.  the stdlib option has to copy things from byte[] at each node.

@FZambia is a bit concerned about use of `unsafe.Pointer` -- but that seems to be how jsoniter is written :man_shrugging:

Reply:
> but that seems to be how jsoniter is written

i think that's how _all_ faster json encode/decode libs for Go are written

Reply:
> @aknuds1 -- ah ok, I avoided init() because of the lint rules, but if that seems like a reasonable option then lets do it.

So this project has a linter to stop the use of `init`, right. Your code there essentially does the same as `init` though. If there's no way around an `init`, I think it's better to use one and silence the linter with a comment.